### ISSUE TITLE
Use the official APIs

### ISSUE THREAD

**Issue by SimoLRepo** [j-fbriere/squawker#46]
### Describe the problem

Hi all, just today I stumbled upon a <a href="https://www.sotwe.com/">front-end for Twitter</a> that (<a href="https://www.sotwe.com/about">as per description</a>) use the same APIs as Twitter.

### Describe the solution

It would be possible to add an option to to choose use either Nitter or the <a href="https://developer.twitter.com/en/docs/twitter-api">Twitter APIs</a>? In this way, <a href="https://github.com/j-fbriere/squawker/issues/45">if Nitter encounters any malfunctions</a>, the user can switch to the official APIs.

**Comment by SimoLRepo** [comment]
@Davilarek Unfortunately, I checked the various types of Twitter API subscription plans after I posted this issue. I thought it was a fairly fixable solution. Obviously there was not said to be a unique account, a specific user would use their own API assigned by Twitter. Also it would have been an optional setting, the important thing is to make the user aware.

Hopefully this issue will be fixed as soon as possible (assuming it is fixable. I wouldn't want something similar to what happened to Aurora Store to happen). From what I understand the error is coming from Nitter. Unfortunately Squawker relying Nitter also carries with it the related technical problems.

Thanks anyway.
